Lombiq.HelpfulLibraries.AspNetCore/Security/CdnContentSecurityPolicyProvider.cs

```diff
@@ -32,6 +32,7 @@ public class CdnContentSecurityPolicyProvider : IContentSecurityPolicyProvider
     public static ConcurrentBag<Uri> PermittedScriptSources { get; } = new(new[]
```

Comment:
While we're at it, let's prevent future such issues by adding all default CDNs used by OC. `https://maxcdn.bootstrapcdn.com` is one too, but check out `OrchardCore.Resources`.

Reply:
These are the unique hosts used in [`OrchardCore.Resources.ResourceManagementOptionsConfiguration`](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs):
- https://cdn.jsdelivr.net/
- https://cdnjs.cloudflare.com/
- https://code.jquery.com/
- https://maxcdn.bootstrapcdn.com/

Added the two highlighted that were missing.